auth-web/src/views/auth/MaintainBusinessView.vue

```diff
@@ -36,6 +36,50 @@
           <learn-more-button
            :redirect-url="learnMoreUrl"
           />
+          <p v-if="!userProfile" class="mt-5">Don't have your Cooperative Passcode?
+            <v-menu top offset-y v-model="contactUsPopover" :close-on-content-click="false" attach="#maintain-info-container">
+              <template v-slot:activator="{ on, attrs }">
+                <span v-bind="attrs" v-on="on" class="popover-activator">Contact Us</span>
+              </template>
+              <v-card class="contact-popover">
+                <v-list class="pa-5" max-width="30rem">
+                  <v-list-item class="pb-3">
+                    <v-list-item-title class="popover-title mt-5 font-weight-bold">Don't have a Passcode for your Cooperative<br>Association?</v-list-item-title>
+                    <v-list-item-action class="mt-0">
+                      <v-icon @click="contactUs = false" color="bcgovblueLink">mdi-close</v-icon>
+                    </v-list-item-action>
+                  </v-list-item>
+                  <v-list-item>
+                    <v-list-item-content>
+                      <v-list-item-subtitle>If you have not recieved your Access Letter from BC Registries,<br>or have lost your Passcode, please contact us at:</v-list-item-subtitle>
+                    </v-list-item-content>
+                  </v-list-item>
+                  <v-list-item>
+                    <v-list-item-content>
+                      <v-list-item-subtitle>Toll Free: 1-877-526-1526</v-list-item-subtitle>
+                      <v-list-item-subtitle class="my-1">Phone: 250 387-7848</v-list-item-subtitle>
+                      <v-list-item-subtitle>Email: <a href="mailto:bcregistries@gov.bc.ca?subject=BC Registries and Online Services - Support Request">bcregistries@gov.bc.ca</a></v-list-item-subtitle>
+                    </v-list-item-content>
+                  </v-list-item>
+                  <v-list-item>
+                    <v-list-item-content>
+                      <v-list-item-subtitle>Hours of Operation:</v-list-item-subtitle>
+                      <v-list-item-subtitle class="my-1">Monday to Friday, 8:30am - 4:30pm PST</v-list-item-subtitle>
+                    </v-list-item-content>
+                  </v-list-item>
+                  <v-list-item>
+                    <v-list-item-content>
+                      <v-list-item-subtitle>
+                        <a href="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/news-updates/modernization/coops-services-card" rel="noopener noreferrer" target="_blank">
```

Comment:
I recommend moving this long link into a property.

Reply:
Done and done, next commit!